### PR TITLE
Fix docs examples.

### PR DESCRIPTION
**Pull request by WiXSL**

### CODE REVIEW

File: docs/Actions.md [hunk 1]
```diff
@@ -141,15 +143,16 @@ In practice, react-admin uses `useQueryWithStore` instead of `useQuery` everywhe
...
+import { useMutation, Button } from 'react-admin';
 
 const ApproveButton = ({ record }) => {
     const [approve, { loading }] = useMutation({
         type: 'update',
         resource: 'comments',
         payload: { id: record.id, data: { isApproved: true } }
     });
-    return <FlatButton label="Approve" onClick={approve} disabled={loading} />;
+    return <Button label='Approve' onClick={approve} disabled={loading} />;
```
Comment (djhi) [inline docs/Actions.md]: Please revert the quotes

File: docs/Actions.md [hunk 2]
```diff
@@ -222,11 +225,12 @@ React-admin provides one hook for each of the Data Provider methods. Based on `u
...
 
 ```jsx
-import { useUpdate } from 'react-admin';
+import React from 'react';
+import { useUpdate, Button } from 'react-admin';
 
 const ApproveButton = ({ record }) => {
     const [approve, { loading }] = useUpdate('comments', record.id, { isApproved: true }, record);
-    return <FlatButton label="Approve" onClick={approve} disabled={loading} />;
+    return <Button label='Approve' onClick={approve} disabled={loading} />;
```
Comment (djhi) [inline docs/Actions.md]: Please revert the quotes

File: docs/Actions.md [hunk 3]
```diff
@@ -270,7 +275,7 @@ const ApproveButton = ({ record }) => {
             notify(`Comment approval error: ${error.message}`, 'warning');
         });
     
-    return <FlatButton label="Approve" onClick={approve} disabled={loading} />;
+    return <Button label='Approve' onClick={approve} disabled={loading} />;
```
Comment (djhi) [inline docs/Actions.md]: Please revert the quotes

File: docs/Actions.md [hunk 4]
```diff
@@ -307,7 +313,7 @@ const ApproveButton = ({ record }) => {
             onFailure: (error) => notify(`Comment approval error: ${error.message}`, 'warning'),
         }
     );
-    return <FlatButton label="Approve" onClick={approve} disabled={loading} />;
+    return <Button label='Approve' onClick={approve} disabled={loading} />;
```
Comment (djhi) [inline docs/Actions.md]: Please revert the quotes

File: docs/Actions.md [hunk 5]
```diff
@@ -343,7 +350,7 @@ const ApproveButton = ({ record }) => {
             onFailure: (error) => notify(`Error: ${error.message}`, 'warning'),
         }
     );
-    return <FlatButton label="Approve" onClick={approve} disabled={loading} />;
+    return <Button label='Approve' onClick={approve} disabled={loading} />;
```
Comment (djhi) [inline docs/Actions.md]: Please revert the quotes

File: docs/Actions.md [hunk 6]
```diff
@@ -370,7 +378,7 @@ const ApproveButton = ({ record }) => {
...
```
Comment (djhi) [inline docs/Actions.md]: Please revert the quotes

File: docs/Actions.md [hunk 7]
```diff
@@ -404,7 +413,7 @@ const ApproveButton = ({ record }) => {
...
```
Comment (djhi) [inline docs/Actions.md]: Please revert the quotes

File: docs/Actions.md [hunk 8]
```diff
@@ -478,24 +490,24 @@ const ApproveButton = ({ record }) => {
...
-            type="update"
-            resource="comments"
+            type='update'
+            resource='comments'
             payload={payload}
             options={options}
         >
             {(approve, { loading }) => (
-                <FlatButton label="Approve" onClick={approve} disabled={loading} />
+                <Button label='Approve' onClick={approve} disabled={loading} />
```
Comment (djhi) [inline docs/Actions.md]: Please revert the quotes

File: docs/Actions.md [hunk 9]
```diff
@@ -577,10 +588,10 @@ const ApproveButton = ({ record }) => {
                 setLoading(false);
                 dispatch(fetchEnd()); // stop the global loading indicator
             });
-    }
+    };
 
-    return <Button label="Approve" onClick={handleClick} disabled={loading} />;
-}
+    return <Button label='Approve' onClick={handleClick} disabled={loading} />;
```
Comment (djhi) [inline docs/Actions.md]: Please revert the quotes

File: docs/Theming.md [hunk 10]
```diff
@@ -410,14 +413,15 @@ export default MyLayout;
...
+import React from 'react';
 import { AppBar, UserMenu, MenuItemLink } from 'react-admin';
 import SettingsIcon from '@material-ui/icons/Settings';
 
 const ConfigurationMenu = forwardRef(({ onClick }, ref) => (
     <MenuItemLink
         ref={ref}
         to="/configuration"
-        primaryText={Configuration}
+        primaryText={"Configuration"}
```
Comment (djhi) [inline docs/Theming.md]: You can even remove the braces

File: docs/Theming.md [hunk 11]
```diff
@@ -961,7 +968,7 @@ Display a linear progress component. Display the same loading component as `reac
 Usage:
 
 ```jsx
-({ data, ...props }) => !data?
+({ data, ...props }) => !data ?
         <LinearProgress /> :
-        <MyInput data={data} />
+        <MyInput data={data} />;
```
Comment (djhi) [inline docs/Theming.md]: I'm pretty sure this semi-colon shouldn't be here

File: docs/Theming.md [hunk 12]
```diff
@@ -970,5 +970,5 @@ Usage:
 ```jsx
 ({ data, ...props }) => !data ?
         <LinearProgress /> :
-        <MyInput data={data} />;
+        <MyInput data={data} />
```
Comment (djhi) [inline docs/Theming.md]: My mistake, the semi-colon was correct...

File: docs/Theming.md (hunk 10 above)
Comment (djhi) [reply docs/Theming.md]: I meant you can replace the braces `{}` with double quotes.

**Comment by WiXSL** [comment]
Done!

**Comment by WiXSL** [comment]
I found a couple more.

**Pull request merged**